.github/workflows/make_dbc_release.yml

```diff
@@ -0,0 +1,74 @@
+
+on:
+  #run on pushes to master
+  push:
+    branches: [ master ]
+
+jobs:
+  make_dbc:
+    #not sure what to put here, referenced main.yml
```

Comment:
Yep this looks good! @abdullahs26 when you get main.yml updated to a higher ubuntu and it's working, could you then upgrade this script?

Reply:
Yeah sure, sounds good.